[Feature] set mpv title

**Describe the bug**

In the MacOS version, the title passed to mpv is the file name not the title of the content.

Rationale: when looking at the race from multiple content windows, it would be convenient to have the name of the content attached to the feed, for example when multiple driver windows are streaming, keeping track of the driver name via the window.

**To Reproduce**

Steps to reproduce the behavior:

Launch f1viewer.
Select Main Feed
mpv application is launched.
The title passed is the path to the content not the content name.

**Expected behaviour**

The "--title" passed to mpv appears as "clip_2_master.m3u8?1234567890&ttl=123456&cc=US&token=123token12312312312 - mpv". 

Is there a way to pass the stream display name:

	"Race Name | Session Name | Main Feed" 
	or 
	"Race Name | Session Name | Driver Name"
	
	mpv //path/to/resource --title="window title"


**Screenshots**

Selecting content in this screenshot:

![image](https://user-images.githubusercontent.com/68130234/87207983-551fdf80-c2db-11ea-9160-7cddd04fe067.png)

Showing the title of the resultant window in this screenshot:

![image](https://user-images.githubusercontent.com/68130234/87207996-610ba180-c2db-11ea-82ee-908de812f80e.png)

**F1TV account plan**
[this section left intentionally blank]

**Desktop (please complete the following information):**

This occurs on 
 -    Mac OS 10.14.6 
running 
-     F1viewer 1.1.0 
-     built 2020-06-29T15:29:14Z 
-     installed via brew
with  
-     mpv 0.32.0
-     built on Tue Jun 16 15:22:16 BST 2020

**Logs**
2020/07/10 18:25:57 [ERROR] Could not access credential store: Specified keyring backend not available
2020/07/10 18:25:57 [ERROR] No keyring configured
2020/07/10 18:25:58 [INFO] checking for live session
2020/07/10 18:26:01 [INFO] no live session found
2020/07/10 18:26:08 [INFO] login successful!
2020/07/10 18:26:14 [INFO] login successful!
2020/07/10 18:26:14 [ERROR] No keyring configured
2020/07/10 18:26:21 [INFO] checking for live session
2020/07/10 18:26:22 [INFO] no live session found
2020/07/10 18:26:42 [INFO] checking for live session
2020/07/10 18:26:45 [INFO] no live session found
2020/07/10 18:27:05 [INFO] checking for live session
2020/07/10 18:27:08 [INFO] no live session found
2020/07/10 18:27:28 [INFO] checking for live session
2020/07/10 18:27:31 [INFO] no live session found

--- a/node.go
+++ b/node.go
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"fmt"
 	"os/exec"
 
 	"github.com/atotto/clipboard"
@@ -58,7 +59,8 @@ func (session *viewerSession) getPlaybackNodes(sessionTitles titles, epID string
 			session.logError(err)
 			return
 		}
-		cmd := exec.Command("mpv", url, "--alang="+session.cfg.Lang, "--start=0", "--quiet")
+
+		cmd := exec.Command("mpv", url, "--alang="+session.cfg.Lang, "--start=0", "--quiet", fmt.Sprintf(`--title="%s"`, sessionTitles.String()))
 		err = session.runCmd(cmd)
 		if err != nil {
 			session.logError(err)

